static/js/components/CourseCard.js

```diff
@@ -41,7 +42,7 @@ const CourseCard = ({
             alt={`cover image for ${course.title}`}
           />
         </div>
-        <div className="row course-title" onClick={showCourseDrawer}>
+        <div className="row course-title" onClick={showResourceDrawer}>
```

Comment:
Is there some reason we need a separate card type for bootcamps? I think if we can do it it would be better to rename this `CourseCard` into `LearningResourceCard` and then modify it to support both courses and bootcamps, since we're displaying basically the same stuff for them. What do you think?

Reply:
They are very similar but some of the info is different - for example bootcamps do not have a `semester` or `platform`.  I initially considered adding some conditional statements for these to `CourseCard` but ultimately decided to create separate card components in case they ever diverged further, and also because the `objectType` parameter sent to `LearningResourceDrawer` from the card needs to be different.  But if you think it's better to do the conditional approach I can change it.

Reply:
I think if they need to diverge further in the future we can create separate components then, but unless we need to do that it's better to add a few small conditionals and avoid writing extra code.